### ISSUE TITLE
need to disable create-default-database.sh and create-visibility-database.sh steps

### ISSUE THREAD

**Issue by bkcsfi** [alexandrevilain/temporal-operator#650]
Hi,

I am trying to setup a temporal cluster with postgres based on cloudnative-pg

The database and user already exists when the temporal operator starts.

However the schema setup process fails with this error:

```
Unable to create SQL database. {"error": "pq: permission denied to create database", "logging-call-at": "handler.go:94"}
```

I believe its failing because the create-default-database.sh  script always attempts to create the database.

I've tried editing the configmap in argocd and k9s to comment out this script, but I can't seem to change it.

Unfortunately the temporal-sql-tool doesn't appear to support a 'create if not exists' option.

Any suggestions on how to get temporal-operator to bypass this step since the databases already exist?

**Comment by bkcsfi** [comment]
I found a workaround.

I reviewed the source for the temporal-sql-tool to see how it handles the create process.  That code is supposed to recognize that the database already exists. However the create step fails because the db user is not authorized to create a database.

My workaround was to change the cloudnative-pg configuration so that the default database user is also a superuser. This is not a great solution, but moves things forward.

I think I will open a case with temporal to see if the temporal-sql-tool can handle this situation better.

thanks
